recipe/build.sh

```diff
@@ -27,21 +31,58 @@ if [[ $target_platform == linux-* ]]; then
   export LDFLAGS="${LDFLAGS} -lrt"
 fi
 
+# Set the vcpkg target triplet otherwise vcpkg can't find the
+# compilers
+if [[ $target_platform == osx-arm64  ]]; then
+  export VCPKG_TARGET_TRIPLET="arm64-osx"
+fi
+if [[ $target_platform == osx-64  ]]; then
+  export VCPKG_TARGET_TRIPLET="x64-osx"
+fi
+if [[ $target_platform == linux-64  ]]; then
+  export VCPKG_TARGET_TRIPLET="x64-linux"
+fi
+if [[ $target_platform == linux-ppc64le  ]]; then
+  export VCPKG_TARGET_TRIPLET="ppc64le-linux"
+fi
+if [[ $target_platform == linux-aarch64  ]]; then
+  export VCPKG_TARGET_TRIPLET="arm64-linux"
+fi
+
+# For snowflake build only, disabling all the ports:
```

Comment:
Does it mean that `abs.yaml` must have `upload_channels: sfe1ed40`? Or will the package be delivered to `defaults`?

Reply:
Yes, I will update `abs.yaml` with destination channel.

Reply:
This version will only be delivered to sfe1ed40, for defaults we will have to build with all the backends

Reply:
done